edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_submit.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This is the end of your proctored exam
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Make sure your responses and work are ready to be submitted. Once they are, you may end the exam below.
```

Comment:
After you submit your exam, your responses are graded and your proctoring session is reviewed.

You might be eligible to earn academic credit for this course if you complete all required  exams as well as achieve a final grade that meets credit requirements for the course.